rules/rbe_repo.bzl

```diff
@@ -872,6 +886,10 @@ def rbe_autoconfig(
       exec_compatible_with: Optional. List of constraints to add to the produced
           toolchain/platform targets (e.g., ["@bazel_tools//platforms:linux"] in the
           exec_compatible_with/constraint_values attrs, respectively.
+      exec_properties: Optional. A string->string dict containing execution properties
```

Comment:
wrap at the same length as all other lines in this file

Reply:
Out of curiosity, what *is* the maximum line length?
It doesn't seem to be the "standard" 80 or 100.

Reply:
I think its 80, but we dont enforce it via any tools, so as long as it looks decent in github side by side view it's good to go

Reply:
done